<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMainWindow</class>
 <widget class="QMainWindow" name="QtMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>771</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mandelbrot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Mandelbrot.png</normaloff>Mandelbrot.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 11px;
    text-align: center;
}
QProgressBar::chunk {
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(6, 69, 163, 255), stop:1 rgba(42, 131, 255, 255));
    border-radius: 10px;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="8">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMandelbrotWidget" name="mandelbrotWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>600</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSaveImage1920x1080"/>
    <addaction name="actionSaveImage2560x1440"/>
    <addaction name="actionSaveImage3840x2160"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuPrecision">
     <property name="title">
      <string>Precision</string>
     </property>
     <addaction name="actionPrecisionAuto"/>
     <addaction name="actionDouble"/>
     <addaction name="actionFixedPoint128"/>
    </widget>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionResetZoom"/>
    <addaction name="separator"/>
    <addaction name="actionAnimatePalette"/>
    <addaction name="separator"/>
    <addaction name="menuPrecision"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionType"/>
    <addaction name="actionTypeMandelbrot"/>
    <addaction name="actionTypeJulia"/>
    <addaction name="actionJuliaSetOptions"/>
    <addaction name="separator"/>
    <addaction name="actionIterations"/>
    <addaction name="actionIterAuto"/>
    <addaction name="actionIter128"/>
    <addaction name="actionIter192"/>
    <addaction name="actionIter256"/>
    <addaction name="actionIter384"/>
    <addaction name="actionIter512"/>
    <addaction name="actionIter768"/>
    <addaction name="actionIter1024"/>
    <addaction name="actionIter1536"/>
    <addaction name="actionIter2048"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/res/icons/about-svgrepo-com.svg</normaloff>:/Icons/res/icons/about-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/res/icons/copy-svgrepo-com.svg</normaloff>:/Icons/res/icons/copy-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionViewOutputs">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/res/icons/image-picture-svgrepo-com.svg</normaloff>:/Icons/res/icons/image-picture-svgrepo-com.svg</iconset>
   </property>
   <property name="text">
    <string>View Outputs</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionResetZoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>...</string>
   </property>
  </action>
  <action name="actionOpenMP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenMP</string>
   </property>
  </action>
  <action name="actionSaveImage1920x1080">
   <property name="text">
    <string>Save Image 1920x1080</string>
   </property>
  </action>
  <action name="actionSaveImage2560x1440">
   <property name="text">
    <string>Save Image 2560x1440</string>
   </property>
  </action>
  <action name="actionSaveImage3840x2160">
   <property name="text">
    <string>Save Image 3840x2160</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="actionAnimatePalette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Animate Palette</string>
   </property>
  </action>
  <action name="actionPrecisionAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionDouble">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Double</string>
   </property>
  </action>
  <action name="actionFixedPoint128">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fixed point 128</string>
   </property>
  </action>
  <action name="actionType">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </action>
  <action name="actionTypeMandelbrot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mandelbrot</string>
   </property>
  </action>
  <action name="actionTypeJulia">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Julia Set</string>
   </property>
  </action>
  <action name="actionJuliaSetOptions">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Julia Set Options</string>
   </property>
  </action>
  <action name="actionIterations">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Iterations</string>
   </property>
  </action>
  <action name="actionIter128">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>128</string>
   </property>
  </action>
  <action name="actionIter192">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>192</string>
   </property>
  </action>
  <action name="actionIter256">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>256</string>
   </property>
  </action>
  <action name="actionIter384">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>384</string>
   </property>
  </action>
  <action name="actionIter512">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>512</string>
   </property>
  </action>
  <action name="actionIter768">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>768</string>
   </property>
  </action>
  <action name="actionIter1024">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1024</string>
   </property>
  </action>
  <action name="actionIter1536">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1536</string>
   </property>
  </action>
  <action name="actionIter2048">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
  </action>
  <action name="actionIterAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMandelbrotWidget</class>
   <extends>QWidget</extends>
   <header location="global">QMandelbrotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>QtMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
